src/cli/onefuzz/api.py

```diff
@@ -692,11 +692,17 @@ class Notifications(Endpoint):
     endpoint = "notifications"
 
     def create(
-        self, container: primitives.Container, config: models.NotificationConfig
+        self,
+        container: primitives.Container,
+        config: models.NotificationConfig,
+        *,
+        replace_existing: bool = True,
```

Comment:
I think the default should be false to prevent accidentally deleting in the case where multiple notification are created on a container

Reply:
Addressed per conversation